content/en-us/docs/developer/run-a-local-development-network.md

```diff
@@ -18,22 +18,52 @@ In this tutorial, we're going to set up a "Hello World" development environment.
 A full Phala Network stack has three components plus a Web UI. The core components are at [Phala-Network/phala-blockchain](https://github.com/Phala-Network/phala-blockchain):
 
 - `phala-node`: The Substrate blockchain node
-- `pRuntime`: The TEE runtime. Contracts runs in pRuntime
-- `phost`: The Substrate-TEE bridge relayer. Connects the blockchain and pRuntime
+- `pRuntime`: The TEE runtime. Contracts runs in `pRuntime`
+- `phost`: The Substrate-TEE bridge relayer. Connects the blockchain and `pRuntime`
```

Comment:
Is this based on an outdated version? Since the latest version here should be `pherry`

Reply:
@shelvenzhou yes this is the outdated version. I am continuously updating it with the current. The newest commit should have this updated to `pherry` 

Just uploaded it here:
https://github.com/hauni97/phala-wiki-next/commit/15fee1a73deb910520b3f18c10d227b3926eae39